tf/ecs/task-definition.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_ecs_task_definition" "main" {
+  family = var.name
+
+  container_definitions = <<CONTAINERS
+[
+  {
+    "name": "api",
```

Comment:
I'm assuming that since this is the name for the _container_ within the task definition that this won't conflict with the `api` container for the API task on the api project?

Reply:
That's correct. There could be more than one container in a task, so that's how you separate them.